llvm/lib/Support/Parallel.cpp

```diff
@@ -57,7 +57,7 @@ class ThreadPoolExecutor : public Executor {
     Threads.resize(1);
     std::lock_guard<std::mutex> Lock(Mutex);
     // Use operator[] before creating the thread to avoid data race in .size()
-    // in “safe libc++” mode.
+    // in "safe libc++" mode.
```

Comment:
```suggestion
    // in 'safe libc++' mode.
```
Single quotes are more common for stuff like this.